extensions/fairifier-rdf-extension/dtls/fairifier/RdfSkeletonJsonTransformerImpl.java

```diff
@@ -1,26 +1,48 @@
+
 package org.dtls.fairifier;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
 
-public class RdfSkeletonJsonTransformerImpl implements RdfSkeletonTransformer{
-    private Model model = new Model();
-    
-    @Override
-    public void transform(String model) {
-        this.model.setJson(model);
-    }
+import com.fasterxml.jackson.databind.ObjectMapper;
 
-    @Override
-    public String getModelAsJsonString() {
-        return this.model.getJson();
-    }
+public class RdfSkeletonJsonTransformerImpl implements RdfSkeletonTransformer {
+
+    private static final ObjectMapper mapper = new ObjectMapper();
+    private Model model;
```

Comment:
This approach is not thread safe. The model can exist solely within the scope of the `read()` and `write()` methods.